lib/service/service.go

```diff
@@ -923,15 +923,23 @@ func initExternalLog(auditConfig services.AuditConfig) (events.IAuditLog, error)
 		case dynamo.GetName():
 			hasNonFileLog = true
 			cfg := dynamoevents.Config{
-				Tablename: uri.Host,
-				Region:    auditConfig.Region,
+				Tablename:               uri.Host,
+				Region:                  auditConfig.Region,
+				EnableContinuousBackups: auditConfig.EnableContinuousBackups,
+				EnableAutoScaling:       auditConfig.EnableAutoScaling,
+				ReadMaxCapacity:         int(auditConfig.ReadMaxCapacity),
+				ReadMinCapacity:         int(auditConfig.ReadMinCapacity),
+				ReadTargetValue:         auditConfig.ReadTargetValue,
+				WriteMaxCapacity:        int(auditConfig.WriteMaxCapacity),
+				WriteMinCapacity:        int(auditConfig.WriteMaxCapacity),
```

Comment:
There is a subtle error here: You're setting the minimum capacity to the maximum capacity value.

Reply:
Other than this, the branch seems to be working fine 👍